src/main/webapp/WEB-INF/jsp/transfer.jsp

```diff
@@ -8,45 +8,52 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
```

Comment:
I did not think so. I copied the code from signup for css. it came with
that.

On Sat, Oct 17, 2015 at 11:24 AM, Nagarjuna Myla <notifications@github.com>
wrote:

> In src/main/webapp/WEB-INF/jsp/transfer.jsp
> <https://github.com/chasethenag420/sbs/pull/59#discussion_r42312396>:
>
> > @@ -8,45 +8,52 @@
> >  <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
> >  <html>
> >  <head>
> > -<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
> > +<meta name="viewport" content="width=device-width, initial-scale=1.0">
> > +<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
>
> Our schema has character encoding as UTF-8 is there any reason to change
> to ISO-8859-1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/59/files#r42312396>.
>



-- 
Thanks and Regards,
*Ganesh R Zilpe*
*Graduate Student(Computer Science)*
*Arizona State University*
*ganesh.zilpe@asu.edu <ganesh.zilpe@asu.edu>  |  +1-480-289-0918*